lib/pure/times.nim

```diff
@@ -489,140 +683,134 @@ proc years*(y: int): TimeInterval {.inline.} =
   ## ``echo getTime() + 2.years``
   initInterval(0,0,0,0,0,0,y)
 
-proc `+=`*(t: var Time, ti: TimeInterval) =
-  ## modifies `t` by adding the interval `ti`
-  t = toTime(getLocalTime(t) + ti)
+proc `+=`*(time: var Time, interval: TimeInterval) =
+  ## Modifies `time` by adding `interval`.
```

Comment:
Use double \`\` for idents.

Reply:
For parameter names single quotes are possible and used throughout Nim's documentation. The documentation generator doesn't have good support for them yet though.

Reply:
The reason I asked for this is because I'm fairly sure the doc gen just adds italics to single quotes.